mslib/mscolab/seed.py

```diff
@@ -32,6 +32,7 @@
 from mslib.mscolab.conf import mscolab_settings
 from mslib.mscolab.models import User, db, Permission, Operation
 from mslib.mscolab.server import APP as app
+from mslib.mscolab.server import content
```

Comment:
you need here as content the STUB_CODE  maybe move it to here

And you need that also as default in the tests for test_server.py